packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.controller.js

```diff
@@ -147,4 +171,31 @@ export default class PciServingNamespaceModelsAddController {
   getTax(id) {
     return this.pricesCatalog[`ai-serving-engine.${id}.hour.consumption`].tax;
   }
+
+  onChangeFramework(framework) {
+    this.model.backend = this.backends.find(
+      (backend) => backend.id === framework.recommendedBackend,
+    );
+  }
+
+  getBackendLabel(backend, framework) {
+    let label = backend.name;
+    if (backend.id === framework.recommendedBackend) {
```

Comment:
I wonder if adding a `oui-badge` next to the recommended choice would be better and suitable in that context